website/routes/AllFeatures.tsx

```diff
@@ -0,0 +1,38 @@
+import { faker } from '@faker-js/faker';
+import { createFileRoute } from '@tanstack/react-router';
+
+import type { Row } from './AllFeatures.lazy';
+
+let rows: readonly Row[] | undefined;
+
+function createRows(): Row[] {
```

Comment:
This still blocks the browser while creating all the rows.

There are a few alternative:
- `scheduler.postTask()`, see below, doesn't work in Firefox yet
  - https://developer.mozilla.org/en-US/docs/Web/API/Scheduler/postTask
- `scheduler.yield()`, not yet available
  - https://developer.chrome.com/blog/chrome-129-beta#scheduleryield
- A web worker
  - https://v3.vitejs.dev/guide/features.html#web-workers

Would be interesting to set up a web worker.

```tsx
    const row = await scheduler.postTask(
      (): Row => ({
        id: `id_${i}`,
        avatar: faker.image.avatar(),
        email: faker.internet.email(),
        title: faker.person.prefix(),
        firstName: faker.person.firstName(),
        lastName: faker.person.lastName(),
        street: faker.location.street(),
        zipCode: faker.location.zipCode(),
        date: faker.date.past().toLocaleDateString(),
        bs: faker.company.buzzPhrase(),
        catchPhrase: faker.company.catchPhrase(),
        companyName: faker.company.name(),
        words: faker.lorem.words(),
        sentence: faker.lorem.sentence()
      })
    );

    rows.push(row);
```